src/progressive-profiling/ProgressiveProfiling.jsx

```diff
@@ -39,47 +40,78 @@ import {
 import { trackRecommendationsGroup, trackRecommendationsViewed } from '../recommendations/track';
 
 const ProgressiveProfiling = (props) => {
+  const { formatMessage } = useIntl();
   const {
-    formRenderState, submitState, showError, location,
+    getFieldDataFromBackend,
+    location,
+    submitState,
+    showError,
+    welcomePageContext,
+    welcomePageContextApiStatus,
   } = props;
+  const queryParams = useMemo(() => getAllPossibleQueryParams(), []);
+  const isRegistrationEmbedded = queryParams?.variant === EMBEDDED;
```

Comment:
please use this util created https://github.com/openedx/frontend-app-authn/blob/d41c06b1fd048572342f05e38af84267e7df82b2/src/data/utils/dataUtils.js#L80